### PR TITLE
pandas tutorial announcement

### PR DESCRIPTION
**Pull request by jukent**
@mgrover1 @dcamron Here is the blog post for the Pandas tutorial announcement.

### CODE REVIEW

File: site/posts/pandas-tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "\n",
+    "The content for this tutorial is hosted on Max's [NCAR Pandas Tutorial Github Repository](https://github.com/mgrover1/ncar_pandas_tutorial)\n",
+    "\n",
+    "## Preparation\n",
+    "\n",
+    "- Check your conda install if this is the first tutorial in the series you are joining us for with `conda --version` and if necessary [install by following these instructions](https://docs.conda.io/en/latest/miniconda.html).\n",
+    "\n",
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "- Navigate to Maxs [NCAR Pandas Tutorial Github Repository](https://github.com/mgrover1/ncar_pandas_tutorial).\n",
```
Comment (dcamron) [inline site/posts/pandas-tutorial.ipynb:18]:
```suggestion
    "- Navigate to Max's [NCAR Pandas Tutorial Github Repository](https://github.com/mgrover1/ncar_pandas_tutorial).\n",
```

File: site/posts/pandas-tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "\n",
+    "- Clone this repository by clicking the green \"Code\" button and copying the SSH address under the word \"Clone\". This can be done by clicking the clipboard icon.\n",
+    "\n",
+    "- In your terminal, type `git clone PASTE` where you paste the SSH address you just copied (`git clone git@github.com:mgrover1/ncar_pandas_tutorial.git`)\n",
+    "\n",
+    "- Create a Conda environment from the `conda_environment.yml` file with the terminal command `conda env create -f conda_environment.yml`. Make sure you are in the local `ncar_pandas_tutorial` directory for this step.\n",
```
Comment (dcamron) [inline site/posts/pandas-tutorial.ipynb:24]: @mgrover1 Let's make sure to name our environment spec appropriately.

**Review by dcamron — CHANGES_REQUESTED** [review]
I might suggest we go ahead and wait until Friday to formally post this as Max and I will be still working on content, environment, and organization. We should be done by Friday afternoon. Don't want people sneaking a peek that gives them a wrong idea, heh. Otherwise an apostrophe, looks good! Thanks for putting this together

File: site/posts/pandas-tutorial.ipynb (hunk 2 above)
Comment (mgrover1) [inline site/posts/pandas-tutorial.ipynb:24]: Going to use environment.yml - keeping it very basic (just pandas and jupyterlab)

**Review by mgrover1 — COMMENTED** [review]
I agree with @dcamron - let's hold off until Friday to post. Sorry for the delay in response - busy with the dask summit this week.

File: site/posts/pandas-tutorial.ipynb (hunk 2 above)
Comment (jukent) [reply site/posts/pandas-tutorial.ipynb:24]: That's fine! Thanks for reviewing it :)

**Review by dcamron — APPROVED** [review]

File: site/posts/pandas-tutorial.ipynb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    "\n",
+    "- Navigate to Max's [NCAR Pandas Tutorial Github Repository](https://github.com/mgrover1/ncar_pandas_tutorial).\n",
+    "\n",
+    "- Clone this repository by clicking the green \"Code\" button and copying the SSH address under the word \"Clone\". This can be done by clicking the clipboard icon.\n",
+    "\n",
+    "- In your terminal, type `git clone PASTE` where you paste the SSH address you just copied (`git clone git@github.com:mgrover1/ncar_pandas_tutorial.git`)\n",
+    "\n",
+    "- Create a Conda environment from the `environment.yml` file with the terminal command `conda env create -f environment.yml`. Make sure you are in the local `ncar_pandas_tutorial` directory for this step.\n",
+    "\n",
+    "- Test your installation in the terminal with `conda activate ncar_pandas_tutorial`. \n",
```
Comment (dcamron) [inline site/posts/pandas-tutorial.ipynb:26]: @mgrover1 noting to make sure we specify the name in line with this

**Comment by jukent** [comment]
@mgrover1 @dcamron Is this ready for merge or are there more edits you'd like to make?

Also we should schedule a practice talk for next Monday or Tuesday.

File: site/posts/pandas-tutorial.ipynb (hunk 3 above)
Comment (mgrover1) [reply site/posts/pandas-tutorial.ipynb:26]: this looks good to me - the names of files + environments match

**Review by mgrover1 — APPROVED** [review]
This is good to go!

**Comment by mgrover1** [comment]
> @mgrover1 @dcamron Is this ready for merge or are there more edits you'd like to make?
> 
> Also we should schedule a practice talk for next Monday or Tuesday.

I will go ahead and send an invite to XDev during the team time on Monday

**Pull request merged**